contrib/platform/build.xml

```diff
@@ -169,16 +169,22 @@ com.sun.jna.platform.wince;version=${osgi.version}
       <property name="results.junit" location="${build}/junit-results/${os.prefix}"/>
       <mkdir dir="${results.junit}"/>
       <echo>Saving test results in ${results.junit}</echo>
-      <condition property="tests.platform" value="**/mac/**/*Test.java">
+      <condition property="tests.platform.mac" value="**/mac/**/*Test.java">
         <os family="mac"/>
       </condition>
-      <condition property="tests.platform" value="**/win32/**/*Test.java">
+      <condition property="tests.platform.windows" value="**/win32/**/*Test.java">
         <os family="windows"/>
       </condition>
-      <condition property="tests.platform" value="**/unix/**/*Test.java">
+      <condition property="tests.platform.linux" value="**/linux/**/*Test.java">
+        <os name="Linux"/>
+      </condition>
+      <condition property="tests.platform.unix" value="**/unix/**/*Test.java">
         <os family="unix"/>
```

Comment:
Please change this line to read:
```xml
      <and>
        <os family="unix"/>
        <not><os family="mac"/></not>
      </and>
```

That way the unix tests are not run on Mac OS (as it was before the change)